example/themes/original/src/components/TaxInformation/index.tsx

```diff
@@ -29,6 +29,9 @@ export const TaxInformation = (props: taxInformationParams) => {
   const [, t] = useLanguage()
 
   const includedOnPriceString = data?.type === 1 ? `(${t('INCLUDED_ON_PRICE', 'Included on price')})` : `(${t('NOT_INCLUDED_ON_PRICE', 'Not included on price')})`
+  const hideProductsSectionOffers = ['offer_target_2', 'offer_target_3'].includes(type)
```

Comment:
Again, separate this array of elements in another const, in order to make validations more scalable